controller/invitation.go

```diff
@@ -126,25 +128,37 @@ func (c *InvitationController) AcceptInvite(ctx *app.AcceptInviteInvitationConte
 		}, "failed to accept invitation, invalid code")
 
 		errResponse := err.Error()
-		redirectURL, err = rest.AddParam(redirectURL, "error", errResponse)
-		ctx.ResponseData.Header().Set("Location", redirectURL)
+		acceptedRedirectURL, err = rest.AddParam(acceptedRedirectURL, "error", errResponse)
+		ctx.ResponseData.Header().Set("Location", acceptedRedirectURL)
 		return ctx.TemporaryRedirect()
 	}
 
-	_, err = c.app.InvitationService().Accept(ctx, acceptCode)
+	_, absoluteSpacePath, err := c.app.InvitationService().Accept(ctx, acceptCode)
 
 	if err != nil {
 		errResponse := err.Error()
-		redirectURL, err = rest.AddParam(redirectURL, "error", errResponse)
+		acceptedRedirectURL, err = rest.AddParam(acceptedRedirectURL, "error", errResponse)
 		if err != nil {
 			return err
```

Comment:
I think we should not just return an error, but rather, that should be a JSON-API error